tools/accuracy_checker/accuracy_checker/metrics/reid.py

```diff
@@ -31,6 +29,32 @@
 PairDesc = namedtuple('PairDesc', 'image1 image2 same')
 
 
+def _average_binary_score(binary_metric, y_true, y_score):
+    def binary_target(y):
+        return not (len(np.unique(y)) > 2) or (y.ndim >= 2 and len(y[0]) > 1)
+
+    if binary_target(y_true):
+        return binary_metric(y_true, y_score, sample_weight=None)
+
+    y_true = y_true.ravel()
+    y_score = y_score.ravel()
+
+    if y_true.ndim == 1:
+        y_true = y_true.reshape((-1, 1))
+
+    if y_score.ndim == 1:
+        y_score = y_score.reshape((-1, 1))
+
+    n_classes = y_score.shape[1]
+    score = np.zeros((n_classes,))
+    for c in range(n_classes):
+        y_true_c = y_true.take([c], axis=1).ravel()
+        y_score_c = y_score.take([c], axis=1).ravel()
+        score[c] = binary_metric(y_true_c, y_score_c, None)
```

Comment:
It would seem that `binary_metric` is always called with `sample_weight=None`, so you could just remove that parameter.

Reply:
removed and from `precision_recall_curve` too